src/pages/Static/About.tsx

```diff
@@ -211,67 +201,64 @@ export default function About() {
           padding="12px 28px"
           as={ExternalLink}
           href={process.env.REACT_APP_DMM_ANALYTICS_URL || `https://dev-dmm-info.knstats.com/`}
-          style={{ width: 'auto', marginTop: '5px', fontSize: '16px' }}
+          style={{ width: 'auto', fontSize: '16px' }}
         >
           Analytics
         </ButtonOutlined>
         <ButtonOutlined
           padding="12px 28px"
           as={ExternalLink}
           href={`https://github.com/dynamic-amm`}
-          style={{ width: 'auto', marginTop: '5px', fontSize: '16px' }}
+          style={{ width: 'auto', fontSize: '16px' }}
         >
           Github
         </ButtonOutlined>
         <ButtonOutlined
           padding="12px 28px"
           as={ExternalLink}
           href={`https://files.kyber.network/DMM-Feb21.pdf`}
-          style={{ width: 'auto', marginTop: '5px', fontSize: '16px' }}
+          style={{ width: 'auto', fontSize: '16px' }}
         >
           Lightpaper
         </ButtonOutlined>
         <ButtonOutlined
           padding="12px 28px"
           as={ExternalLink}
           href={`https://docs.dmm.exchange`}
-          style={{ width: 'auto', marginTop: '5px', fontSize: '16px' }}
+          style={{ width: 'auto', fontSize: '16px' }}
         >
           Developer Support
         </ButtonOutlined>
       </div>
 
-      <Text
-        fontSize={36}
-        color={'#f4f4f4'}
-        mt={200}
-        mb={45}
-        width={'100%'}
-        maxWidth={'700px'}
-        marginLeft={'auto'}
-        marginRight={'auto'}
-      >
+      <Text fontSize={[24, 36]} color={'#f4f4f4'} mt={[100, 200]} mb={45} px={2}>
         Committed to Security
       </Text>
       <div className={style.security}>
         <div>
-          <Text fontSize={18}>Code Audited</Text>
+          <Text fontSize={[12, 18]} fontWeight={500}>Code Audited</Text>
           <img src={require('../../assets/svg/about_icon_insurance.svg')} />
         </div>
         <div>
-          <Text fontSize={18}>On-chain and Open Source</Text>
+          <Text fontSize={[12, 18]} fontWeight={500}>On-chain and Open Source</Text>
           <img src={require('../../assets/svg/about_icon_github.jpg')} />
         </div>
         <div>
-          <Text fontSize={18}>Bug Bounty</Text>
+          <Text fontSize={[12, 18]} fontWeight={500}>Bug Bounty</Text>
           <img src={require('../../assets/svg/about_icon_bug_bounty.svg')} />
         </div>
+        <div>
+          <Text fontSize={[12, 18]} fontWeight={500}>Insured by</Text>
+          <img src={require('../../assets/svg/unslashed.svg')} />
```

Comment:
as I remember, we are not allowed to add Unslashed logo previously. Please ask marketing team to see if we can add it now

Reply:
Yes, It's removed